Essentia/UserCases/Login/Controllers/WelcomeViewController.swift

```diff
@@ -61,36 +62,29 @@ class WelcomeViewController: BaseViewController, ImportAccountDelegate, SelectAc
                              .underlineStyle: NSUnderlineStyle.single.rawValue]
             )
         )
-        termsButton.setAttributedTitle(termsAttributedText, for: .normal)
-        
-        // MARK: - Colors
-        title1Label.textColor = colorProvider.appTitleColor
-        title2Label.textColor = colorProvider.appTitleColor
-        termsButton.titleLabel?.textColor = colorProvider.appLinkTextColor
         
-        // MARK: - Font
-        title1Label.font = AppFont.regular.withSize(36)
-        title2Label.font = AppFont.bold.withSize(36)
+        return [.attributedCenteredButton(attributedTitle: termsAttributedText, action: termsAction, textColor: colorProvider.appLinkTextColor, background: .clear),
```

Comment:
If all parameters not fit in one line then paste every parameter in signature from new line

Reply:
Same on all lines > 120

Reply:
fixed, ef08e75